### PR TITLE
More fixes for webfont loading

### PR DESCRIPTION
**Pull request by johnspackman**
fixes some bugs in font loading: order of libraries is preserved; font-face src is corrected; manifest fix;

### CODE REVIEW

**Review by oetiker — APPROVED** [review]

**Review by hkollmann — CHANGES_REQUESTED** [review]
Should´nt foliojs-fork/fontkit in line 71 removed then?

**Comment by goldim** [comment]
@johnspackman thank you for replacing dependency. I also wanted to make a PR where the fork was replaced by original library. And seems like original one now more fresh then fork which supposed to be newer than original (in fork description said that original library hasn't been updated during long time). Btw tests are successful on my local machine.

File: source/class/qx/tool/cli/commands/ExportGlyphs.js [hunk 1]
```diff
@@ -48,9 +48,12 @@ qx.Class.define("qx.tool.cli.commands.ExportGlyphs", {
           /^font\/(ttf|svg|eot|woff|woff2)$/
         );
       }
+      /*
```
Comment (goldim) [inline source/class/qx/tool/cli/commands/ExportGlyphs.js:51]: Should it be left as commented?

**Comment by goldim** [comment]
@johnspackman there is one more usage of the fork in file `qx.tool.compiler.app.WebFont`

**Review by goldim — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Comment by hkollmann** [comment]
@johnspackman: Could you have a loók at the conflicts and can you replace the package-lock.json?

**Comment by goldim** [comment]
Also some tests are still failed.

**Comment by hkollmann** [comment]
@johnspackman : Any idea what happens with test-cli tests?

**Review by cboulanger — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]
I don't use mobile so trusting you here

**Review by derrell — APPROVED** [review]

**Pull request merged**